packages/beacon-node/src/network/processor/gossipHandlers.ts

```diff
@@ -563,7 +564,11 @@ function getSequentialHandlers(modules: ValidatorFnsModules, options: GossipHand
         chain.getBlobsTracker.triggerGetBlobs(blockInput);
         // if we've received at least half of the columns, trigger reconstruction of the rest
         if (blockInput.columnCount >= NUMBER_OF_COLUMNS / 2) {
-          chain.columnReconstructionTracker.triggerColumnReconstruction(blockInput);
+          chain.columnReconstructionTracker.triggerColumnReconstruction(
+            // wait to reconstruct until after head vote
+            getCutoffTimeMs(chain, dataColumnSlot, config.SECONDS_PER_SLOT / INTERVALS_PER_SLOT),
```

Comment:
last arg should be in ms
```suggestion
            getCutoffTimeMs(chain, dataColumnSlot, (config.SECONDS_PER_SLOT / INTERVALS_PER_SLOT) * 1000),
```